spec/support/project_hepler.rb

```diff
@@ -0,0 +1,6 @@
+def add_project
+  visit root_path
+  click_link 'Projects'
+  fill_in :project_name, with: 'Project'
+  click_button 'Add project!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.